examples/cpp/desktop-testbed/1_build_desktop.sh

```
@@ -0,0 +1,3 @@
+clear
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)